internal/venice-common/src/main/resources/avro/IngestionTaskReport/v1/IngestionTaskReport.avsc

```diff
@@ -21,6 +21,11 @@
       "type": "boolean",
       "default": true
     },
+    {
+      "name": "exceptionThrown",
+      "type": "boolean",
+      "default": false
+    },
```

Comment:
So this schema is always guaranteed to be the same between parent and child processes, and therefore does not need versioning, correct...?

Reply:
Yes, this is the good part for evolving schema in II compared to other protocols.